src/query/atomWithQuery.ts

```diff
@@ -98,16 +98,19 @@ export function atomWithQuery<
         const unsubscribe = observer.subscribe(listener)
         return unsubscribe
       }
-      return { dataAtom, options }
+      return {
+        dataAtom,
+        observer,
+      }
```

Comment:
```suggestion
      return { dataAtom, observer }
```
Does prettier complain with this?

Reply:
this is auto formatted via prettier

Reply:
let me try.